examples/Microsoft.Spark.CSharp.Examples/Sql/VectorUdfs.cs

```diff
@@ -8,6 +8,9 @@
 using Microsoft.Spark.Sql;
 using Microsoft.Spark.Sql.Types;
 using StructType = Microsoft.Spark.Sql.Types.StructType;
+using FxDataFrame = Microsoft.Data.DataFrame;
```

Comment:
@imback82 @rapoth - do either of you have any thoughts/opinions on how to make this better? Using 2 types named `DataFrame` in Spark seems unfortunate. Is there a better term we can call the corefx `DataFrame` class?